audio.go

```diff
@@ -5,23 +5,53 @@ import (
 	"math"
 )
 
-type FramesInt []int
+type SamplesInt []int
```

Comment:
Note, it seems dangerous to me to use `int` as the base type. This would end up different conversions from float64 to int based on the platform. I would think that either int32  or int64 here would be a better option overall.